.github/workflows/pull-requests-test-and-lint.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       - name: Check linting on all files
         run: npm run lint
 
+      - name: Stylelint Owner Fix
+        run: sudo chown -R root:root $GITHUB_WORKSPACE
```

Comment:
woah what's happening here? Is there docs showing that this needs to be done?

Reply:
Based on this comment - https://github.com/reviewdog/reviewdog/issues/1158#issuecomment-1097649732

Reply:
gotcha cool